graphql_client_cli/src/main.rs

```diff
@@ -38,9 +41,21 @@ enum Cli {
     #[structopt(name = "generate")]
     Generate {
         // should be a glob
-        paths: String,
+        /// Path to graphql query file
         #[structopt(parse(from_os_str))]
-        schema: PathBuf,
+        query_path: PathBuf,
+        /// Path to graphql schema file
+        #[structopt(parse(from_os_str))]
+        schema_path: PathBuf,
+        /// Name of struct
```

Comment:
I have ideas on how we might change this in the future (because we can basically generate code for all operations, so I just opened issue #134. It will be a big refactoring and it needs to be discussed so let us merge this PR first :)